src/disk.c

```diff
@@ -1217,20 +1238,20 @@ static int disk_read(void) {
       continue;
 
     metric_t m = {0};
-    metric_label_set(&m, "device", drives[i].name);
+    metric_label_set(&m, device_label, drives[i].name);
 
-    metric_family_append(&fam_io, "direction", "read",
+    metric_family_append(&fam_io, direction_label, read_direction,
                          (value_t){.counter = drives[i].rbytes}, &m);
-    metric_family_append(&fam_io, "direction", "write",
+    metric_family_append(&fam_io, direction_label, write_direction,
                          (value_t){.counter = drives[i].wbytes}, &m);
 
-    metric_family_append(&fam_ops, "direction", "read",
+    metric_family_append(&fam_ops, direction_label, read_direction,
                          (value_t){.counter = drives[i].rxfer}, &m);
-    metric_family_append(&fam_ops, "direction", "write",
+    metric_family_append(&fam_ops, direction_label, write_direction,
                          (value_t){.counter = drives[i].wxfer}, &m);
 
-    m.value.counter = drives[i].time_sec * 1000 + drives[i].time_usec / 1000;
-    metric_family_metric_append(&fam_disk_io_time, m);
+    m.value.counter = drives[i].time_sec + drives[i].time_usec / 1000000;
+    metric_family_metric_append(&fam_ops_time, m);
```

Comment:
This metric family name change does not correspond with changes at start of the `disk_read()`.

Reply:
Not sure I understand you correctly. Did you mean that this is in seconds while other systems report in us? I have changed all implementations to report us.

Reply:
I mean that this change is:
* `fam_disk_io_time` -> `fam_ops_time`

Whereas other changes in the file were:
* `fam_disk_io_time` ->  `fam_disk_io_time`  (no fam name change)
* `fam_time` -> `fam_ops_time`